src/api.ts

```diff
@@ -25,4 +25,18 @@ function get<T>(url: string): Promise<T> {
 
 export const getTodos = () => get<Todo[]>('/todos');
 
+export const getAll = () => getTodos().then(todos => todos);
+export const getActive = () =>
+  getTodos().then(response => response.filter(todo => !todo.completed));
+
+export const getCompleted = () =>
+  getTodos().then(response => response.filter(todo => todo.completed));
+
+export const getFounded = (text: string) =>
+  getTodos().then(response =>
+    response.filter(todo =>
+      todo.title.toLowerCase().includes(text.toLowerCase()),
+    ),
+  );
```

Comment:
it is redundant to load todos from server every time.
load once and filter on client